packages/api/src/platforms/vtex/index.ts

```diff
@@ -92,16 +92,16 @@ const Resolvers = {
 
 export const getContextFactory =
   (options: Options) =>
-    (ctx: any): Context => {
-      ctx.storage = {
-        channel: ChannelMarshal.parse(options.channel),
-        flags: options.flags ?? {},
-        locale: options.locale,
-      }
-      ctx.clients = getClients(options, ctx)
-      ctx.loaders = getLoaders(options, ctx)
-
-      return ctx
+  (ctx: any): Context => {
```

Comment:
This is a formatting error, right? It's the second time I've seen this in unrelated PRs, I think the linter might be getting confused here.